test/helpers/cons.go

```diff
@@ -156,7 +156,9 @@ const (
 
 	// CurlConnectTimeout is the timeout for the connect() call that curl
 	// invokes
-	CurlConnectTimeout = 5
+	// FIXME: Revert this back to 5 seconds once the IPCache propagation delay
+	// (which seems to take up to 4-8 seconds) has been addressed.
```

Comment:
[Nit] Can we add a link to a tracking issue?

Reply:
Done!